{{cookiecutter.project_name}}/Dockerfile

```
@@ -1,7 +1,7 @@
 FROM python:{{ cookiecutter.python_version }}-slim-bookworm
 
 # Install uv
-COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
+COPY --from=ghcr.io/astral-sh/uv:{{ cookiecutter._docker-uv-version }} /uv /uvx /bin/
```

Changing to snake case for consistency:

```suggestion
COPY --from=ghcr.io/astral-sh/uv:{{ cookiecutter._docker_uv_version }} /uv /uvx /bin/
```